src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -716,41 +759,41 @@ void func_80A75C38(EnIk* this, GlobalContext* globalCtx) {
     }
 
     if (this->actor.colChkInfo.health == 0) {
-        func_80A7598C(this);
+        EnIk_SetupDeath(this);
         Enemy_StartFinishingBlow(globalCtx, &this->actor);
         return;
     }
     Math_SmoothStepToS(&this->actor.world.rot.y, this->actor.yawTowardsPlayer, 1, 0x7D0, 0);
     if ((this->actor.params == 0) && (Rand_ZeroOne() < 0.5f)) {
         if (ABS((s16)(this->actor.yawTowardsPlayer - this->actor.shape.rot.y)) > 0x4000) {
-            func_80A754A0(this);
+            EnIk_SetupSingleHorizontalAttack(this);
         }
     }
-    if ((this->actor.params != 0) && (this->unk_2FB != 0)) {
+    if ((this->actor.params != 0) && (this->armorStatusFlag != 0)) {
         if ((prevHealth > 10) && (this->actor.colChkInfo.health <= 10)) {
             Audio_PlayActorSound2(&this->actor, NA_SE_EN_IRONNACK_ARMOR_OFF_DEMO);
         } else {
             Audio_PlayActorSound2(&this->actor, NA_SE_EN_IRONNACK_DAMAGE);
             Audio_PlayActorSound2(&this->actor, NA_SE_EN_NUTS_CUTBODY);
         }
-        func_80A75790(this);
+        EnIk_SetupReactToAttack(this);
         return;
     }
     Audio_PlayActorSound2(&this->actor, NA_SE_EN_IRONNACK_ARMOR_HIT);
     Audio_PlayActorSound2(&this->actor, NA_SE_EN_IRONNACK_DAMAGE);
     CollisionCheck_SpawnShieldParticles(globalCtx, &sp38);
 }
 
-void func_80A75FA0(Actor* thisx, GlobalContext* globalCtx) {
+void EnIk_AxeAttackHitPlayer(Actor* thisx, GlobalContext* globalCtx) {
```

Comment:
This function is the update function while fighting iron knuckles, I would name it `UpdateSomething` (and possibly `EnIk_Update` -> `EnIk_UpdateCutscene`)

and similarly with `EnIk_DrawParamType` and `EnIk_Draw`

Reply:
bump